sycl/test/on-device/plugins/interop-level-zero.cpp

```diff
@@ -0,0 +1,50 @@
+// REQUIRES: level_zero
```

Comment:
I think we should make "REQUIRES: level_zero" mean both runtime & headers are available (currently only runtime is checked).

Reply:
I am not sure this check is needed. Level Zero headers are always copied to the include directory during the build. So we can always compile the test. We do the same for opencl and don't check for headers. 
Even if you insist on this I think this check should be implemented for both opencl and level zero together and it is out of scope of this PR.